upcoming-release-notes/3028.md

```diff
@@ -0,0 +1,6 @@
+---
+category: Enhancements
+authors: [matt-fidd]
+---
+
+Add mergePayees method to the API
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏spellchecker will later complain about this if we don't wrap it in code block
```suggestion
Add `mergePayees` method to the API
```